biorxiv/pca_association_experiment/biorxiv_pca_plots.py

```diff
@@ -0,0 +1,639 @@
+
+# ### PCA1 vs PCA4
+
+# In[19]:
+
+
+display_clouds(
+    'output/word_pca_similarity/figures/pca_01_cossim_word_cloud.png',
+    'output/word_pca_similarity/figures/pca_04_cossim_word_cloud.png'
+)
+
+
+# These word cloud produces the following concepts: qunatitative biology vs molecular biology (left) and marine biology vs cell biology (right).
+
+# In[20]:
+
+
+g = (
+    p9.ggplot(pca_df)
+    + p9.aes(x="pca1", y="pca4", color="factor(category)")
+    + p9.geom_point()
+    + p9.labs(
+        title="PCA of BioRxiv (Word Dim: 300)",
+        color="Article Category"
+    )
+)
+g.save("output/pca_plots/pca01_v_pca04.png", dpi=500)
+print(g)
+
+
+# In[21]:
+
+
+(
+    pca_df
+    [["pca1", "pca4", "category", "doi"]]
+    .query("pca1 < -2 & pca4 > 0")
+    .category
+    .value_counts()
+    .head(10)
+)
+
+
+# In[22]:
+
+
+(
+    pca_df
+    [["pca1", "pca4", "category", "doi"]]
+    .query("pca1 < 0 & pca4 < 0")
+    .category
+    .value_counts()
+    .head(10)
+)
+
+
+# Looking on the left size of PCA1 (negative), it looks like the top quadrant contains mainly cell biology papers and variants of cell biology. The bottom quadrant contains papers that are related to microbiology; Fun anecdote is that I super convinced that PCA4 was marine biology, but upon closer inspection turns out I was wrong it is microbio.
+
+# # Non-straightforward PCs
+
+# This section of the notebook aims to take a look at PCs that are not as straightforward as the ones above.
+
+# ## PCA1 vs PCA6
+
+# In[23]:
+
+
+display_clouds(
+    'output/word_pca_similarity/figures/pca_01_cossim_word_cloud.png',
+    'output/word_pca_similarity/figures/pca_06_cossim_word_cloud.png'
+)
+
+
+# The right word cloud appears to represent mathematics vs scientific communication or at least popular buzz words scientist used to promote their research. The next few cells will look more into it.
+
+# In[24]:
+
+
+g = (
+    p9.ggplot(pca_df)
+    + p9.aes(x="pca1", y="pca6", color="factor(category)")
+    + p9.geom_point()
+    + p9.labs(
+        title="PCA of BioRxiv (Word Dim: 300)",
+        color="Article Category"
+    )
+)
+g.save("output/pca_plots/pca01_v_pca06.png", dpi=500)
+print(g)
+
+
+# In[25]:
+
+
+(
+    pca_df[["pca1", "pca6", "category", "doi"]]
+    .drop_duplicates("doi")
+    .query("pca1 > 0 & pca6 > 3")
+    .category
+    .value_counts()
+    .head(10)
+)
+
+
+# In[26]:
+
+
+(
+    pca_df[["pca1", "pca6", "category", "doi"]]
+    .drop_duplicates("doi")
+    .query("pca1 > 0 & pca6 < -1.5")
+    .category
+    .value_counts()
+    .head(10)
+)
+
+
+# Looking at the top categories for the top and bottom right quadrants it seems that the papers follow the patterns captures by the word clouds above; however the positive axis still remains difficult to judge without taking a look at the individual papers.
```

Comment:
Looks like bioengineering, chemistry are positive which I would've expected to be more mathematical..but its a little hard for me to tell from the plot.